### PR TITLE
feat(dx): Add command for turning on translation markers

### PR DESCRIPTION
**Pull request by evanpurkhiser**
convert

### CODE REVIEW

File: src/sentry/static/sentry/app/locale.jsx [hunk 1]
```diff
@@ -178,15 +187,15 @@ function mark(rv) {
...
+      className: markerCss,
       children: _.isArray(rv) ? rv : [rv],
     },
     _owner: null,
     _store: {},
   };
 
   proxy.toString = function() {
-    return '🇦🇹' + rv + '🇦🇹';
+    return '✅' + rv + '✅';
```
Comment (EvanPurkhiser) [inline src/sentry/static/sentry/app/locale.jsx]: The red seemed to imply a bit too much 'missing translation' to me.

File: src/sentry/static/sentry/app/components/search/sources/commandSource.jsx [hunk 2]
```diff
@@ -25,6 +26,13 @@ const ACTIONS = [
       }),
   },
 
+  {
+    title: 'Toggle Translation Markers',
+    description: 'Toggles translation markers on or off in the application',
```
Comment (EvanPurkhiser) [inline src/sentry/static/sentry/app/components/search/sources/commandSource.jsx]: Ironically this should be translated, but the others aren't, so we'll come back for these later.

File: src/sentry/static/sentry/app/locale.jsx [hunk 3]
```diff
@@ -12,14 +14,21 @@ if (sessionStorage && sessionStorage.getItem('localeDebug') == '1') {
   LOCALE_DEBUG = true;
 }
 
+const markerCss = css`
+  background: #ff801790 !important;
```
Comment (billyvg) [inline src/sentry/static/sentry/app/locale.jsx]: Do we need `!important` since you removed the css?

**Review by billyvg — APPROVED** [review]

File: src/sentry/static/sentry/app/locale.jsx (hunk 3 above)
Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/locale.jsx]: hm, probably not actually

File: src/sentry/static/sentry/app/locale.jsx (hunk 1 above)
Comment (dcramer) [reply src/sentry/static/sentry/app/locale.jsx]: Rip Austria

Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/locale.jsx]: oh it's a flag lol, I thought it was like, the NO symbol.

image

I think my emoji font makes it unclear.

File: src/sentry/static/sentry/app/components/search/sources/commandSource.jsx [hunk 4]
```diff
@@ -25,6 +26,13 @@ const ACTIONS = [
...
+    requiresSuperuser: true,
+    action: () => toggleLocaleDebug(),
```
Comment (dcramer) [inline src/sentry/static/sentry/app/components/search/sources/commandSource.jsx]: Can’t you just return the function

**Review by dcramer — APPROVED** [review]
Seems mostly ok

File: src/sentry/static/sentry/app/components/search/sources/commandSource.jsx (hunk 4 above)
Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/components/search/sources/commandSource.jsx]: Sure. The other one is like this tho, so I'll go back and get those after merging.

**Pull request merged**